third_party/txt/src/skia/paragraph_builder_skia.cc

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "paragraph_builder_skia.h"
+#include "paragraph_skia.h"
+
+#include "third_party/skia/modules/skparagraph/include/ParagraphStyle.h"
+#include "third_party/skia/modules/skparagraph/include/TextStyle.h"
+#include "txt/paragraph_style.h"
+
+namespace skt = skia::textlayout;
+
+namespace txt {
+
+namespace {
+
+SkFontStyle MakeSkFontStyle(txt::FontWeight font_weight,
+                            txt::FontStyle font_style) {
+  return SkFontStyle(static_cast<SkFontStyle::Weight>(
+                         static_cast<int>(font_weight) * 100 + 100),
```

Comment:
I'd pull out the magic numbers into named variables to better document them.

Reply:
Added a comment about this